.rubocop.yml

```diff
@@ -7,3 +7,6 @@ Style/FrozenStringLiteralComment:
 
 Style/Documentation:
   Enabled: false
+
+Metrics/MethodLength:
+  Max: 20
```

Comment:
Do we have such long methods already? Lets try to keep it short for now

Reply:
Yes we have.
lib/bddsm/progress_report.rb:18:5: C: Metrics/MethodLength: Method has too many lines. [11/10]

Reply:
lets add a todo, it would be great to fix it later